README.md

```diff
@@ -1,3 +1,38 @@
 # weather_app
-With this weather app, users can search for the weather forecast for different locations. They will be able to choose whether they want to see the current weather, the hourly forecast for the next 24 hours or a daily forecast for the next 7 days. 
-The project will be done in R Shiny. 
+
+What will you implement first?
+* I will first try to get weather data using the API by https://openweathermap.org.
+
+What is the goal?
+* The goal is to create a weather app where users can get weather forecasts for a location of their choice. They can choose to get information about the current weather, the hourly weather forecast for the next 24 hours, or the daily weather forecast for the next 7 days.  
+
+What do you cut if you’re short on time?
+* I could cut the possibility that users choose between the current weather, an hourly forecast and a daily forecast. Then, the users would always get the current weather and an hourly forecast for the next 24 hours.
+
+What do you extend if you have extra time?
+* I will try to implement a map of the specified location with symbols on the map that are corresponding to the weather forecast for this specific location.   
+* I could try to implement my own function that gets the latitude and longitude of a location (instead of using the package tidygeocoder)
+
+
+
+*The code:*
+Will you use R or Python?
+* I will use R and R Shiny
+
+Which packages will you use?
+* I will use the packages httr, jsonlite, dplyr, shiny, tidygeocoder
+
+Which functions will you create? (A rough flowchart, nothing too detailed)
+* First, the user opens the Shiny App.
+* Then, they type in the location they are interested in and they choose what kind of weather forecast they would like to have (current weather, hourly forecast, daily forecast, there will be a drop down menu for the options).
+* The user press the button "search" to start the search for the weather data.
+* Then, the function tidygeocoder::geo_osm() will search for the longitude and latitude of this location and save it in an output file.
```

Comment:
Good idea to implement functions like those yourself! Reducing dependencies is often a good practice (many dependencies tend to complicate the maintenance of a project). 

[`geo_osm`](https://github.com/jessecambon/tidygeocoder/blob/5032a234c31fe0b3070152bb6e3deca97efc244c/R/geo_osm.R) is a good candidate for this because it looks like it's just a convenience wrapper around `tmaptools::geocode_OSM`. Like you indicate yourself, I'd only do this if you have extra time.